src/main/kotlin/kweb/html/ElementReader.kt

```diff
@@ -19,15 +19,15 @@ open class ElementReader(protected val receiver: WebBrowser, internal val jsExpr
         }
     }
 
-    val tagName: CompletableFuture<String> get() = receiver.evaluate("$jsExpression.tagName").thenApply { it.toString() }
-    val attributes: CompletableFuture<Map<String, Any>> get() = receiver.evaluate("$jsExpression.attributes").thenApply { it as Map<String, Any> }
-    fun attribute(name: String): CompletableFuture<Any> = receiver.evaluate("($jsExpression.getAttribute(\"${name.escapeEcma()}\"));")
+    val tagName: CompletableFuture<String> get() = receiver.callJsFunctionWithResult("return $jsExpression.tagName").thenApply { it.toString() }
```

Comment:
Should we be using jsExpression rather than passing an id as a parameter here and on following lines?